manifests/init.pp

```diff
@@ -5,14 +5,28 @@
 # by an example created by Rackspace's cloudbuilders
 #
 class drbd(
-  $service_enable = true
-) {
+  $service_enable   = true,
+  $version          = $::drbd::params::version,
+  $package_name     = $::drbd::params::package_name,
+  $manage_repo      = $::drbd::params::manage_repo,
+  $replication_mode = 'C',
```

Comment:
I'm having second thoughts about `replication_mode`...

resource.pp has a `protocol` parameter that defaults to 'C'.  Maybe it should default to `undef` instead?